os/net/netmgr/netdev_mgr_internal.c

```diff
@@ -46,7 +46,7 @@ static int g_eth_idx = 0;
  * and application doesn't require write operation to netdev
  * so if there is no write operation then protection wouldn't be required
  */
-sem_t g_netdev_lock = SEM_INITIALIZER(1);
+sem_t g_netdev_lock;
```

Comment:
Is it required only in these 2 places? I observed in some stm files as well. How about PTHREAD_MUTEX_INITIALIZER?

Reply:
Yes, I checked PTHREAD_MUTEX_INITIALIZER also.
There are 2 places more in stm and esp files. I will modify them.